packages/package_info/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.1
```

Comment:
Should be 0.4.0+2 since this package is pre 1.0.0
https://www.dartlang.org/tools/pub/versioning#semantic-versions

Reply:
done